### ISSUE TITLE
`option.rev` should support array of revision.

### ISSUE THREAD

**Issue by dinoboff** [alessioalex/git-commits#2]
e.g.
```
const gitCommits = require('git-commits');

const stream = gitCommits(repoPath, {rev: ['^v1.1.1', 'v1.1.2^@']});
```

This range is not possible with a dotted notation AFAIK.

**Comment by alessioalex** [comment]
Awesome stuff man, I'll check out the branch tomorrow and if everything's ok I'll merge it in and publish. Thanks for contributing!

**Comment by alessioalex** [comment]
@dinoboff merged and published new version on npm: 1.3.0

Sorry for the huge delay, got cought up with some personal stuff. I've added you as a contributor on Github. Let me know if you'd like to have rights to publish on npm as well.
